packages/playground-ui/src/ds/components/Header/Header.tsx

```diff
@@ -2,18 +2,22 @@ import clsx from 'clsx';
 import React from 'react';
 
 import { Txt } from '../Txt';
+import { cn } from '@/lib/utils';
 
 export interface HeaderProps {
   children?: React.ReactNode;
   border?: boolean;
+  className?: string;
 }
 
-export const Header = ({ children, border = true }: HeaderProps) => {
+export const Header = ({ children, border = true, className }: HeaderProps) => {
   return (
     <header
-      className={clsx('h-header-default z-50 flex w-full items-center gap-[18px] bg-transparent px-5', {
-        'border-b-sm border-border1': border,
-      })}
+      className={cn(
```

Comment:
you can go back to clsx. We'll manage tailwind inconsitencies in a case by case scenario. I think adding the `shrink-0` you did is not an issue and it would simply the tw merge config stuff